### ISSUE TITLE
Error on serde v1.0.211

### ISSUE THREAD

**Issue by OscarHedeby** [VirusTotal/yara-x#229]
Error in dependency with the newest version of serde https://github.com/serde-rs/serde/issues/2844.

```
error[E0425]: cannot find value `__deserializer` in this scope
   --> /github/home/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cargo-0.83.0/src/cargo/core/features.rs:759:32
    |
759 |     #[serde(deserialize_with = "deserialize_build_std")]
    |                                ^^^^^^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find value `__deserializer` in this scope
   --> /github/home/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cargo-0.83.0/src/cargo/core/features.rs:770:32
    |
770 |     #[serde(deserialize_with = "deserialize_git_features")]
    |                                ^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find value `__deserializer` in this scope
   --> /github/home/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cargo-0.83.0/src/cargo/core/features.rs:772:32
    |
772 |     #[serde(deserialize_with = "deserialize_gitoxide_features")]
    |                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in this scope
```

A possible fix would be to pin the serde library to v1.0.210 until the issue is fixed.

**Comment by russellbanks** [comment]
@plusvic This means I can't build my project using the latest version of serde:
```
error: failed to select a version for `serde`.                                                                                                                                                                                                                                                                                                                                                                                   
    ... required by package `yara-x v0.11.0`
    ... which satisfies dependency `yara-x = "^0.11.0"` of package
versions that meet the requirements `=1.0.210` are: 1.0.210

all possible versions conflict with previously selected packages.

  previously selected package `serde v1.0.215`
    ... which satisfies dependency `serde = "^1.0.215"` of package

failed to select a version for `serde` which could resolve this conflict
```

Following the linked issue (and subsequent fixes - https://github.com/serde-rs/serde/pull/2845, https://github.com/serde-rs/serde/pull/2847), it looks like this bug was been resolved in v1.0.213.

**Comment by plusvic** [comment]
@russellbanks I've upgraded to serde 1.0.215 I will bump the minor version of `yara-x`
